src/lib/services/client-metrics/schema.test.ts

```diff
@@ -0,0 +1,99 @@
+import { clientRegisterSchema, clientMetricsSchema } from './schema';
+
+test('clientRegisterSchema should allow empty ("") instanceId', () => {
+    const { value } = clientRegisterSchema.validate({
+        appName: 'test',
+        instanceId: '',
+        strategies: ['default'],
+        started: Date.now(),
+        interval: 100,
+    });
+    //@ts-ignore
```

Comment:
Do we need `//@ts-ignore` here? If so, how about `@ts-expect-error` instead? Same for the rest of the file.

Reply:
no we dont!